.github/workflows/ci.yaml

```diff
@@ -767,6 +767,27 @@ jobs:
           set -euxo pipefail
           kubectl --namespace coder rollout restart deployment/coder
           kubectl --namespace coder rollout status deployment/coder
+  
+  deploy-wsproxies:
+    runs-on: ubuntu-latest
+    needs: build
+    if: !github.event.pull_request.head.repo.fork
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+
+      - name: Deploy workspace proxies
+        run: |
+          flyctl deploy --image ghcr.io/coder/coder-preview:main --app paris-coder -y
+          flyctl deploy --image ghcr.io/coder/coder-preview:main --app sydney-coder -y
+          flyctl deploy --image ghcr.io/coder/coder-preview:main --app sao-paulo-coder -y
```

Comment:
Does this automatically re-pull? a la `imagePullPolicy: Always`?

Reply:
Good catch. No it does not unless the tag changes
I am changing it to `--image ghcr.io/coder/coder-preview:$tag`